---
layout: "post"
title: "DSB 2024 Montpellier - March 14-15"
description: Workshop Data Structures in Bioinformatics
permalink: /DSB2024/
image: "mtp_julien_daga_J5iirAB17TA.jpg"
---

## Where and when

DSB 2024 will be organized in Montpellier, South of France

on the Campus St Priest, from Montpellier University

on March 14-15 in 2024.

The workshop  will take place in the Amphitheatre of Building #2 of the Campus (see map below).

- GPS coordinates: 43,6351948 ; 3,8414903
- [Location view on OpenStreetMap](https://www.openstreetmap.org/#map=18/43.6351948/3.8414903 "Workshop Venue Location")
- [View Location on Google Maps](https://maps.app.goo.gl/96Sb9vwir7RE9uih6 "Workshop Venue Location")
- Local map with building #2 entrance highlighted. An door to the campus is just 15 m away.
  The nearest Tram station is "Chateau d'O", located at the roundabout of the same name.
  ![Local map](./stpriest_map_bat2.png) 

## Program

The detailed program is available as pdf file: [DSB program](./dsb_2024_detailed_prog.pdf "DSB 2024 program")

The list of selected abstracts is below:
| Speaker               | Title                                                                                                             |
|-----------------------+-------------------------------------------------------------------------------------------------------------------|
| Avila Jorge           | A compact embedding-based indexing for accurate and rapid classification in bacterial pangenomics                 |
| Rivals Eric           | An incremental algorithm for computing the set of all period sets                                                 |
| Renders Luca          | Automated design of efficient search schemes for lossless approximate pattern matching                            |
| Smith Caleb           | Brisk: Exact resource-efficient dictionary for k-mers                                                             |
| Martayan Igor         | Conway-Bromage-Lyndon (CBL): an exact, dynamic representation of k-mer sets                                       |
| Wirtz Johannes        | Counting multiple-merger tree structures emerging in Population Genetics                                          |
| Schmitz Johanna       | EpiSegMix: Discovering chromatin states using a flexible distribution hidden Markov model with duration modeling  |
| Rizzo Nicola          | Exploiting uniqueness: seed-chain-extend alignment with Elastic Founder Graphs                                    |
| Biagi Elena           | Finimizers: Variable-length bounded-frequency minimizers for k-mer sets                                           |
| Uzuner Hamdiye        | Generalized uncertainty-aware haplotype quantification with application in HLA typing and virus analysis.         |
| Luca Parmigiani       | Interpolating and Extrapolating Node Counts in Colored Compacted de Bruijn Graphs for Pangenome Growth Comparison |
| Hendrychová Veronika  | Mathematical model of phylogenetic compression                                                                    |
| Romashchenko Nikolai  | Memory-frugal disk-based (phylo-)k-mer filtering for alignment-free phylogenetic placement                        |
| Draesslerová Dominika | Metagenomic classification with maximal exact matches in KATKA kernels and minimizer digests                      |
| Rouzé Timothé         | Pangenomic k-mer distribution with low memory cost                                                                |
| Sgro Mattia           | PlasBin-flow on Pangenome graphs: improving bacterial plasmid binning in short-read assemblies                    |
| Depuydt Lore          | r-indexing without backward searching                                                                             |
| Bonnet Konstantinn    | Strangepg: Toward Pangenome Scale Graph Visualization                                                             |
| Levallois Victor      | The Backpack Quotient Filter: a dynamic and space-efficient data structure for querying k-mers with abundance.    |
| Vandamme Lea          | Tinted de Bruijn Graphs for efficient read extraction from sequencing datasets                                    |
| Dubois Siegfried      | Towards an edit distance between pangenome graphs                                                                 |
| Chikhi Rayan          | Work in progress for peta-scale sequence exploration                                                              |


## Organizers
From team "Methods & Algorithms for Bioinformatics":
- Jordan Moutet (LIRMM - CNRS et Univ. Montpellier)
- [Eric Rivals](https://www.lirmm.fr/~rivals/)  (LIRMM - CNRS et Univ. Montpellier)
- Nikolai Romashchenko (LIRMM - CNRS et Univ. Montpellier)
- Pengfei Wang (LIRMM - CNRS et Univ. Montpellier)

and the LIRMM Communication unit
- Virginie Fêche 
- Elena Demchenko

For any questions about DSB2024, please reach out to [rivals@lirmm.fr](mailto:rivals@lirmm.fr).

## Sponsors

Lots of thanks for the financial support of:

1. The ITN network [ALPACA](https://alpaca-itn.eu/ "ALPACA webpage") Marie-Curie
2. The french CNRS research network for bioinformatics: the [GDR BIM](https://www.gdr-bim.cnrs.fr/ "GDR BIM wepbage")

  ![ALPACA logo](./alpaca-logo-m.png)      ![GDR BIM logo](./gdrbim-logo.png)

## How to participate

Registration for participating to DSB 2024 is free and now closed.

Registrations and submissions are now open at: [https://dsb-2024.sciencesconf.org/](https://dsb-2024.sciencesconf.org/ "dsb registration site")

Deadline for registration: February 20th, 2024.

## Abstract submission

You can submit an abstract (max 300 words) via the Sciencesconf system at
 [https://dsb-2024.sciencesconf.org/](https://dsb-2024.sciencesconf.org/ "dsb submission site")

To do so, you will be required to register by creating an account with your professional email.

 **Abstract submission deadline (postponed): February ~~1st~~ 10th, 2024**.

As previously, there is no formal review. We intend to give everyone the opportunity to present but the organizers reserve the right to select talks from the submissions to obtain a diverse and interesting program.

## Information about Montpellier

Most tourist attractions, historic museums, cultural hubs, tasty restaurants are around the city center of Montpellier. Staying around this area. For instance, near the Place de la Comédie, is a good option.  For more travelling information about Montpellier, the following link could be helpful: <https://www.montpellier-france.com/>


### Hotels in Montpellier, France
Some  suggestions for hotels:

-  Hôtel du [Palais](https://www.hoteldupalais-montpellier.fr/)
   3 rue du Palais des Guilhem - 34000 Montpellier
   +33 (0)4 67 60 47 38
   mail: hoteldupalaismontpellier@gmail.com
-  Hôtel du [Parc](https://www.hotelduparc-montpellier.com/)
   8 rue achille bégé 34090 Montpellier France
   04 67 41 16 49
   mail: contact@leparc34.fr
-  Hôtel Les [Troènes](https://www.hotel-les-troenes.fr/)
   near Universités & des Hopitaux
   17 avenue Emile Bertin-sans, à Montpellier
   04.67.04.07.76.
   hotel-les-troenes@wanadoo.fr
-  Best Western Hotel [Comedie Saint-Roch](https://www.bestwestern.fr/fr/hotel-Montpellier-Best-Western-Plus-Hotel-Comedie-Saint-Roch-93831)
   6 & 8 rue Jules Ferry, 34000, Montpellier France 
   Tel : +33 04 99 13 33 44
   Email : contact@lhotel-montpellier.com
-  Hôtel des [Arceaux](https://www.hoteldesarceaux.com/en/)
   33-35, boulevard des arceaux, 34000 Montpellier
   +33 (0)4 67 92 03 03
   <https://www.hoteldesarceaux.com/en/>


### Credits
Photo de Julien Daga sur [Unsplash](https://unsplash.com/fr/photos/un-groupe-de-personnes-debout-au-sommet-dun-escalier-J5iirAB17TA?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash)

